spec/outputs/http_spec.rb

```diff
@@ -1 +1,232 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/http"
+require "thread"
+require "sinatra"
+
+PORT = rand(65535-1024) + 1025
```

Comment:
yeah, i thought @jordansissel https://github.com/jordansissel/ruby-flores does that already, no?